server/values.yaml

```diff
@@ -34,6 +34,30 @@ serviceAccount:
 clustermode: ""
 activeactive: ""
 
+# Hashicorp Vault for secrets management
+# Below vaultSecret and vaultAnnotations section helps setting vault sidecar/init-container agents to load secrets securely
+vaultSecret:
+  enabled: false          # Enable to true once you have secrets in vault and annotations are enabled to load admin and db passwords from vault
+  vaultFilepath: ""       # Change the path to "/vault/secrets/<filename>" as per the setup
+
+# Add hashicorp Vault annotations to enable sidecar/init-container vault agent to load admin and db passwords
+# example annotations for self-hosted vault server:
+vaultAnnotations:
+  ####
+  # vault.hashicorp.com/agent-inject: "true"
+  # vault.hashicorp.com/agent-inject-status: update
+  # vault.hashicorp.com/agent-pre-populate-only: 'false'                 # Enable to true to add vault agent as init-container without sidecar
+  # vault.hashicorp.com/role: "aqua-enforcer"                            # Specify your role used by vault agent auto-auth
```

Comment:
please update aqua-enforcer/enforcer with aqua-server/server

Reply:
@KoppulaRajender and @qtasali we will merge your changes and make a required changes